src/astronomer/azext_astronomer/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
if it's a preview version, `azext.isPreview`; otherwise, the version should be `1.0.0`